Source/d_deh.c

```diff
@@ -46,6 +46,7 @@
 #include "../win32/win_fopen.h"
 #endif
 
+//static boolean bfgcells_modified = false;
```

Comment:
What's the matter with this ominous commented variable?

Reply:
I forgot to remove it.